web/modules/custom/server_style_guide/templates/components/person-card.html.twig

```diff
@@ -0,0 +1,38 @@
+<div class="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-6 p-6 bg-gray-50">
```

Comment:
This isn't aligned with the rest of the codebase - I'd expect to see composition and re-use of other existing/new elements.